cl_manycore/libraries/bsg_manycore_driver.cpp

```diff
@@ -434,6 +434,10 @@ int hb_mc_fifo_transmit (uint8_t fd, hb_mc_fifo_tx_t type, const hb_mc_packet_t
 	// Write 1 to the Transmit Complete bit to clear it
 	hb_mc_fifo_set_ixr_bit(fd, dir, HB_MC_MMIO_FIFO_ISR_OFFSET, HB_MC_MMIO_FIFO_IXR_TC_BIT);
 
+	#ifdef DEBUG
```

Comment:
there are print macros for doing this thing in `bsg_manycore_printing.h`. please use them.